diff --git a/src/http/ngx_http_request.h b/src/http/ngx_http_request.h
index 6198d7e..575bc69 100644
--- a/src/http/ngx_http_request.h
+++ b/src/http/ngx_http_request.h
@@ -355,6 +355,8 @@ struct ngx_http_request_s {
     ngx_http_event_handler_pt         read_event_handler;
     ngx_http_event_handler_pt         write_event_handler;
 
+    void                             *filter_cache;
+
 #if (NGX_HTTP_CACHE)
     ngx_http_cache_t                 *cache;
 #endif
